airflow/api_connexion/schemas/task_instance_schema.py

```diff
@@ -129,13 +129,22 @@ class SetTaskInstanceStateFormSchema(Schema):
 
     dry_run = fields.Boolean(dump_default=True)
     task_id = fields.Str(required=True)
-    execution_date = fields.DateTime(required=True, validate=validate_istimezone)
+    execution_date = fields.DateTime(validate=validate_istimezone)
+    dag_run_id = fields.Str()
```

Comment:
Yeah. I thought of that but why I used dag_run_id is just to be consistent with already existing endpoints which uses `dag_run_id` instead of `run_id`. If it's not a problem, let me know, so I can use it.

See https://airflow.apache.org/docs/apache-airflow/stable/stable-rest-api-ref.html#operation/get_dag_run and https://airflow.apache.org/docs/apache-airflow/stable/stable-rest-api-ref.html#tag/TaskInstance etc